### PR TITLE
Feat : Add Responsive Image Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Responsive Image
I have Added a Responsive image Component in the Component Library. Aqua_UI has two types of Responsive Images.
1. Square Image
2. Rounded Image

### CODE REVIEW

File: Components/Image.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                </a>
+                <a href="/Components/Toast.html">
+                    <li class="sidebar__item">Toast</li>
+                </a>
+            </ul>
+        </div>
+
+        <!-- Responsive Image -->
+        <!-- Responsive Intro -->
+        <div class="Image-Rs card">
```
Comment (tusharkandpal) [inline Components/Image.html:119]: stick to one naming standard please

**Review by tusharkandpal — CHANGES_REQUESTED** [review]

File: Components/Image.html (hunk 1 above)
Comment (shivampandey0) [reply Components/Image.html:119]: +1

File: assets/Css/ComponentsStyles.css [hunk 2]
```diff
@@ -387,4 +387,19 @@ main {
...
 }
 
+/* Responsive Image */
+
+.image-Responsive--demo{
+  width: 100%;
+  height: 100%;
+}
+.image-Responsive--bg{
+  max-width: 30rem;
```
Comment (shivampandey0) [inline assets/Css/ComponentsStyles.css:397]: maybe width: 100% can take care of responsiveness, why max-width: 30rem; here?

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions

File: Components/Image.html (hunk 1 above)
Comment (SaurabhChawda) [reply Components/Image.html:119]: I have used BEM naming Convention but In the Case of Image-Rs, it stands for Image Responsive. Should I change the name?

File: assets/Css/ComponentsStyles.css (hunk 2 above)
Comment (SaurabhChawda) [reply assets/Css/ComponentsStyles.css:397]: Yes, 100% will work but max-width is only for the size of the image.

File: Components/Image.html (hunk 1 above)
Comment (tusharkandpal) [reply Components/Image.html:119]: just use small case what i meant

Comment (SaurabhChawda) [reply Components/Image.html:119]: Okay, I will update it. ThankYou

**Pull request merged**